app/assets/src/components/utils/SampleFields.js

```diff
@@ -0,0 +1,44 @@
+export const SAMPLE_FIELDS = [
+  { name: "name", label: "Name" },
+  { name: "project_name", label: "Project" },
+  { name: "sample_location", label: "Location" },
+  {
+    name: "created_at",
+    label: "Upload Date",
+    parser: val => {
+      return <Moment fromNow date={val} />;
```

Comment:
does <Moment> work even though it's not in this file?

Reply:
thanks for catching, fixed